src/mm/checkpoints.c

```diff
@@ -38,6 +38,81 @@
 #include <statistics.h>
 #include <autockpt.h>
 
+#include <incremental_state_saving.h>
+
+
+
+bool chunks_not_used(malloc_area **m_area, int *bitmap_blocks) {
```

Comment:
* why a pointer of pointer?
* better name
```suggestion
bool check_not_used_chunk_and_clean(malloc_area *m_area, int *bitmap_blocks) {
```